Makefile

```diff
@@ -41,12 +41,15 @@ clean: ## Make a clean source tree
 	rm -rf $$(find . -name *.lock)
 	rm -rf $$(find . -name package-lock.json)
 	rm -rf $$(find . -name .pytest_cache)
-
-# Prepares Elyra for build/packaging/installation
-yarn-install:
-	rm -f yarn.lock package-lock.json
 	yarn cache clean
-	yarn
+	pip uninstall -y jupyterlab-git
+	- jupyter labextension uninstall --no-build @jupyterlab/toc
+	$(call UNLINK_LAB_EXTENSION,application)
```

Comment:
Could we have an `unlink` target and make `clean` depend on it. Do we need to have `install` also depend on it in case of the user only issuing a `make install` ?

Reply:
I did have a separate target but I didn't like the order it was executing so I switched back to all of it in one target. I have no string opinions on it though and can separate them again.

As for make install, I can test some more edge cases but I don't believe make install needs to depend on clean anymore, I believe that was a build issue in 1.x that was fixed, but again I'd need to do some more edge case testing to know for sure. @vabarbosa if you're willing to help test the edge cases since you've also been working on some of this.

Reply:
count me in